Tests/test_file_libtiff.py

```diff
@@ -756,10 +747,9 @@ def check_write(libtiff: bool) -> None:
             with Image.open(out) as reloaded:
                 assert icc_profile == reloaded.info["icc_profile"]
 
-        libtiffs = []
+        libtiffs = [False]
         if Image.core.libtiff_support_custom_tags:
             libtiffs.append(True)
```

Comment:
This can also use `pytest.mark.parametrize`

Reply:
Done